### ISSUE TITLE
Message status help page. Images of text don't have accurate alt text

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#52]
## Description 
Some text is missing from the alt text to entirely describe the image. 
![image](https://user-images.githubusercontent.com/5230720/135672993-b6e3ef89-6984-4324-8cb3-b1e23c0b6f1e.png)

## Potential fix
Consider cropping the image or adding more description to the alt text.

**Comment by amazingphilippe** [comment]
https://notification.canada.ca/messages-status

**Comment by EliseKa** [comment]
Images need to be updated in [Figma](https://www.figma.com/file/OGKKT0ikOSXbKolXSf6yP6/Message-delivery-status-|-État-de-livraison-des-messages?node-id=2%3A952) to include the search bar. And then we need to update the images on the [website](https://notification.canada.ca/messages-status) with those.

@YedidaZalik and I wrote [updated alt text](https://docs.google.com/document/d/1tZzv-52DztlIO4blKVipLyhO-0vT5S6ZQETxnA4PKUI/edit#heading=h.8w04ax4y38c) for the updated images that resolve this issue as well.

**Comment by EliseKa** [comment]
Alright, here are the images that should be used on the https://notification.canada.ca/messages-status page. Not sure what format works best, you can export them in another format in [Figma](https://www.figma.com/file/OGKKT0ikOSXbKolXSf6yP6/Message-delivery-status-|-État-de-livraison-des-messages?node-id=821%3A34).

English:
![Message-status-image-EN](https://user-images.githubusercontent.com/45772213/136091745-0d67c1e3-497c-42f7-a328-0f73828666e5.png)

French:
![Message-status-image-FR](https://user-images.githubusercontent.com/45772213/136091775-fca68693-c3ff-4887-ba8d-e71ecdc1b129.png)
